app/web/features/profile/view/About.tsx

```diff
@@ -34,34 +28,35 @@ const useStyles = makeStyles((theme) => ({
 export default function About({ user }: AboutProps) {
   const classes = useStyles();
```

Comment:
I'd suggest move everything in the `profile` namespace instead - it doesn't belong in global since it's specific to this feature area.